### PR TITLE
Quic style varints

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

**Comment by ekr** [comment]
@hannestschofenig @chris-wood

File: draft-ietf-tls-ctls.md [hunk 1]
```diff
@@ -95,19 +95,30 @@ with varints.
...
+encoding format is borrowed from {{?QUIC=I-D.ietf-quic-transport}},
+and is reproduced below for easy reference.
+
+The variable-length integer encoding reserves the two most significant bits
+of the first byte to encode the base 2 logarithm of the integer encoding length
+in bytes.  The integer value is encoded on the remaining bits, in network byte
+order.
+
+This means that integers are encoded on 1, 2, 4, or 8 bytes and can encode 6,
+14, 30, or 62 bit values respectively.  {{integer-summary}} summarizes the
```
Comment (bifurcation) [inline draft-ietf-tls-ctls.md:107]:
```suggestion
This means that integers are encoded on 1, 2, 4, or 8 bytes and can encode 6-,
14-, 30-, or 62-bit values respectively.  {{integer-summary}} summarizes the
```

File: draft-ietf-tls-ctls.md [hunk 2]
```diff
@@ -95,19 +95,30 @@ with varints.
...
+
+| 2Bit | Length | Usable Bits | Range                 |
+|:-----|:-------|:------------|:----------------------|
+| 00   | 1      | 6           | 0-63                  |
+| 01   | 2      | 14          | 0-16383               |
+| 10   | 4      | 30          | 0-1073741823          |
+| 11   | 8      | 62          | 0-4611686018427387903 |
+{: #integer-summary title="Summary of Integer Encodings"}
+
+For example, the eight byte sequence c2 19 7c 5e ff 14 e8 8c (in hexadecimal)
```
Comment (bifurcation) [inline draft-ietf-tls-ctls.md:118]:
```suggestion
For example, the eight-byte sequence c2 19 7c 5e ff 14 e8 8c (in hexadecimal)
```

**Review by bifurcation — APPROVED** [review]
LGTM, with nits

**Comment by ekr** [comment]
@chris-wood I guess we should maybe ask the WG?

**Comment by chris-wood** [comment]
Probably wouldn't hurt, yeah.

**Comment by bemasc** [comment]
Per #35, I think this should be closed.

**Comment by ekr** [comment]
We decided not to do this.

**Pull request closed without merging**